src/css.js

```diff
@@ -303,7 +316,8 @@ jQuery.extend( {
 			num = parseFloat( val );
 			return extra === true || isFinite( num ) ? num || 0 : val;
 		}
-		return val;
+
+		return val === undefined ? undefined : jQuery.trim( val );
```

Comment:
> Testing in chrome returned the computed values with a preceding space in the string. I will look into cleaning this up.

That's probably as it should be. CSS variables are designed to be very powerful, you should e.g. be able to polyfill new CSS constructs by using them + a JS polyfill. Because of all that, almost everything is a valid value for such a property.

In particular, AFAIK everything between a colon and a semi-colon is the value for the custom property. So if you have the following in your CSS:
```css
div {
    --prop: value;
}
```
then ` value` (with a space) is the correct value. We shouldn't trim it. In 99% of cases it doesn't matter, if you later use `val(--prop)` then the space usually becomes irrelevant but there may be cases where it's expected.

I'd even add that we should have tests ensuring that we don't trim anything. :)